autopilot-daemon/network/iperf3_stop_servers.py

```diff
@@ -0,0 +1,65 @@
+from iperf3_utils import *
+
+
+def kill_all_iperf_servers():
+    try:
+        result = subprocess.run(
+            ["ps", "aux"], text=True, capture_output=True, check=True
+        )
+    except subprocess.CalledProcessError as e:
+        print(f"Error occurred while listing processes: {e}")
+        sys.exit(1)
+
+    processes = result.stdout.splitlines()
+
+    for process in processes:
+        try:
+            if "iperf3 -s" in process and "-s" in process:
+                parts = process.split()
+                if len(parts) > 1:
+                    pid = int(parts[1])
+                    # Not killing default iperf server spun up on entrypoint...
```

Comment:
We can take that server out of the way for good.. meaning from the start cmd

Reply:
Oh? Brilliant. Thanks